parserx.py

```diff
@@ -79,7 +99,16 @@ def generate_output(n):
             print("Invalid. Enter input and output file names separated by a space.")
         except FileNotFoundError:
             print("Input file not found. Please try again.")
-
+    elif n == 3:
+        results = {}
+        results['parsed'] = SigParser().parse(" ".join(sys.argv[3:]))
+        results['inferred'] = SigParser().infer(results['parsed'], ndc=sys.argv[2]) 
+        print(json.dumps(results, indent=4))
+    elif n == 3:
```

Comment:
Please change this to n == `4`

I think this is just a typo.

Reply:
Good catch!